templates/repo/issue/new_form.tmpl

```diff
@@ -38,21 +38,35 @@
 			{{template "repo/issue/branch_selector_field" .}}
 
 			<input id="label_ids" name="label_ids" type="hidden" value="{{.label_ids}}">
-			<div class="ui {{if and (not .Labels) (not .OrgLabels)}}disabled{{end}} floating jump select-label dropdown">
+			<div class="ui {{if not .IsIssueWriter}}disabled{{end}} floating jump select-label dropdown">
 				<span class="text">
 					<strong>{{.i18n.Tr "repo.issues.new.labels"}}</strong>
-					{{svg "octicon-gear" 16}}
+					{{if .IsIssueWriter}}
+						{{svg "octicon-gear" 16}}
+					{{end}}
 				</span>
 				<div class="filter menu" data-id="#label_ids">
-					<div class="no-select item">{{.i18n.Tr "repo.issues.new.clear_labels"}}</div>
-					{{range .Labels}}
-						<a class="{{if .IsChecked}}checked{{end}} item has-emoji" href="#" data-id="{{.ID}}" data-id-selector="#label_{{.ID}}"><span class="octicon-check {{if not .IsChecked}}invisible{{end}}">{{svg "octicon-check" 16}}</span><span class="label color" style="background-color: {{.Color}}"></span> {{.Name}}
-						{{if .Description }}<br><small class="desc">{{.Description}}</small>{{end}}</a>
+					<div class="header" style="text-transform: none;font-size:16px;">{{.i18n.Tr "repo.issues.new.add_labels_title"}}</div>
+					{{if or .Labels .OrgLabels}}
+					<div class="ui icon search input">
+						<i class="search icon"></i>
+						<input type="text" placeholder="...">
```

Comment:
These should all be translatable strings that say "Filter Label", "Filter Milestone" etc..